test/controllers/offer.controller.test.js

```diff
@@ -316,6 +316,137 @@ describe('Offer Controller', () => {
+              .end((err, res) => {
+                expect(res).to.have.status(412);
+                expect(res.body.success).to.be.eql(false);
+                expect(res.body.message).to.be.eql('Validation Error');
+                expect(res.body.error).to.be.eql(`"Date Due" must be larger than or equal to 14`);
+                done();
+              });
+          });
+        });
+
+        context('when deductible refund percentage is outside range', () => {
+          it('returns an error', (done) => {
+            const offer = OfferFactory.build({ otherTerms: { deductibleRefundPercentage: 11 } });
+            request()
+              .post('/api/v1/offer/create')
+              .set('authorization', vendorToken)
+              .send(offer)
+              .end((err, res) => {
+                expect(res).to.have.status(412);
+                expect(res.body.success).to.be.eql(false);
+                expect(res.body.message).to.be.eql('Validation Error');
+                expect(res.body.error).to.be.eql(
+                  `"Deductible Refund Percentage" must be less than or equal to 10`,
+                );
+                done();
+              });
+          });
+        });
+
+        context('when grace period is outside range', () => {
+          it('returns an error', (done) => {
+            const offer = OfferFactory.build({ otherTerms: { gracePeriod: 11 } });
+            request()
+              .post('/api/v1/offer/create')
+              .set('authorization', vendorToken)
+              .send(offer)
+              .end((err, res) => {
+                expect(res).to.have.status(412);
+                expect(res.body.success).to.be.eql(false);
+                expect(res.body.message).to.be.eql('Validation Error');
+                expect(res.body.error).to.be.eql(
+                  `"Grace Period" must be larger than or equal to 30`,
+                );
+                done();
+              });
+          });
+        });
+
+        context('when termination interest is outside range', () => {
+          it('returns an error', (done) => {
+            const offer = OfferFactory.build({ otherTerms: { terminationInterest: 11 } });
+            request()
+              .post('/api/v1/offer/create')
+              .set('authorization', vendorToken)
+              .send(offer)
+              .end((err, res) => {
+                expect(res).to.have.status(412);
+                expect(res.body.success).to.be.eql(false);
+                expect(res.body.message).to.be.eql('Validation Error');
+                expect(res.body.error).to.be.eql(
+                  `"Termination Interest" must be less than or equal to 10`,
+                );
+                done();
+              });
+          });
+        });
+
+        context('when termination period is outside range', () => {
+          it('returns an error', (done) => {
+            const offer = OfferFactory.build({ otherTerms: { terminationPeriod: 211 } });
```

Comment:
```suggestion
            const offer = OfferFactory.build({ otherTerms: { terminationPeriod: 181 
```

Lets use 181 here. I am afraid that if we mistakenly update this to a larger number in the future, something like `200`, the test would still pass which is not the expected behaviour.